Backend/DataStructures.py

```diff
@@ -1,64 +1,96 @@
+    def __init__(self, uid, userid, cont, loc, likes, rtweet, clicks, views, date):
+        self.tweet_id = uid
+        self.user_id = userid
+        self.content = cont
+        self.location = loc
+        self.likes = likes
+        self.retweets = rtweet
+        self.clicks = clicks
+        self.views = views
+        self.tweet_date = date
+
+# end Tweet class
 
 
 # [User class]
 class User:
 
-    def __init__(self, uid, uname, uemail):
-        self.userId = uid
-        self.userName = uname
-        self.userEmail = uemail
+    def __init__(self, uid, uname, signdate):
+        self.user_id = uid
+        self.user_name = uname
+        self.signup_date = signdate
 
-    def get_id(self):
-        return self.userId
+# end User class
 
-    def set_id(self, uid):
-        self.userId = uid
 
-    def get_username(self):
-        return self.userName
+# [Followers class]
+class Followers:
 
-    def set_username(self, uname):
-        self.userName = uname
+    def __init__(self, uid1, uid2, folDate):
+        self.user1_id = uid1
+        self.user2_id = uid2
+        self.follow_date = folDate
 
-    def get_email(self):
-        return self.userEmail
+# end Followers class
 
-    def set_email(self, uemail):
-        self.userEmail = uemail
-# end User class
 
+# [TweetsWithTags class]
+class TweetsWithTag:
 
-class Tweet:
+    def __init__(self, tagid, tweetid, tagname):
+        self.hashtag_id = tagid
+        self.tweet_id = tweetid
+        self.hashtag_name = tagname
+
+# end class TweetsWithTag
+
+
+# [Hashtag class]
+class Hashtag:
+
+    def _init_(self, tagid, tagname, freq):
+        self.hashtag_id = tagid
+        self.hashtag_name = tagname
+        self.frequency = freq
+
+# end class Hashtag
+
+
+# [LocationMap class]
+class LocationMap:
+
+    def __init__(self, locid, city, state):
+        self.location_id = locid
+        self.city = city
+        self.state = state
 
-    def __init__(self, uid, userid, postid, body):
-        self.tweetId = uid
-        self.userId = userid
-        self.postId = postid
-        self.content = body
+# end class LocationMap
 
-    def get_id(self):
-        return self.tweetId
 
-    def set_id(self, uid):
-        self.tweetId = uid
+# [TweetsByLocation class]
+class TweetsByLocation:
 
-    def get_userId(self):
-        return self.userId
+    def __init__(self, loc, tweetid):
+        self.location = loc
+        self.tweet_id = tweetid
 
-    def set_userId(self, uid):
-        self.userId = uid
+# end class TweetsByLocation
 
-    def get_postId(self):
-        return self.postId
 
-    def set_postId(self, pid):
-        self.postId = pid
+# [TrendsByLocation class]
+class TrendsByLocation:
 
-    def get_content(self):
-        return self.content
+    def __init__(self, loc, tagname):
+        self.location = loc
+        self.tag_name = tagname
 
-    def set_content(self, body):
-        self.content = body
\ No newline at end of file
+# end class TrendsByLocation
```

Comment:
none of these classes' corresponding tables exist anymore in the database design in the Design Document II **_Google Doc_**. Please either just delete them all or replace them with classes corresponding to the new tables.